packages/resizable-panels/src/utils/test-utils.ts

```
@@ -0,0 +1,65 @@
+	let didCatch = false;
+	try {
+		callback();
+	} catch (error) {
+		caughtError = error;
+		didCatch = true;
+	} finally {
+		console.error = originalError;
+		console.warn = originalWarn;
+
+		if (didCatch) {
+			throw caughtError;
```

Biome (2.5.7)

[error] Unsafe usage of 'throw'.

(lint/correctness/noUnsafeFinally)